### PR TITLE
[Merged by Bors] - docs: update connector readme

### PR DESCRIPTION
**Pull request by morenol**
Just an update with the readme of the connectors

### CODE REVIEW

**Review by EstebanBorai — APPROVED** [review]
Just minor nit!

File: embeds/connectors/outbound/http.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ fluvio cloud connector create --config ./config.yaml
 Check connector logs by running
 
 ```bash
-fluvio cloud connector logs httpbin
+fluvio cloud connector logs <NAME>
```
Comment (morenol) [inline embeds/connectors/outbound/http.md:83]:
```suggestion
fluvio cloud connector logs httpbin
```

[the remark this answers is not in the document]
Comment (morenol) [reply embeds/connectors/outbound/http.md]: updated and created PR in connector: https://github.com/infinyon/http-sink-connector/pull/24

**Comment by tjtelan** [comment]
The connector docs should be updated in their hub packages, because we have automation to generate the changes to the website. I see that the Kafka docs are different than what is in this PR, and that's going to make it difficult to know where to look for the most up to date references

**Pull request closed without merging**